docs/source/CompilationFlowVerification.ipynb

```diff
@@ -2,6 +2,7 @@
  "cells": [
   {
    "cell_type": "markdown",
+   "metadata": {},
```

Comment:
The changes throughout this file seem to arise from the execution of the Jupiter script. That does not seem intentional? could you please revert the changes in this file?

Reply:
Did that with `nb-clean` nice tool. Might be worth using in a pre-commit hook

Reply:
Is there a pre-commit check for that?